apps/taxes/src/modules/avatax/order-confirmed/avatax-order-confirmed-payload-transformer.ts

```diff
@@ -42,6 +40,11 @@ export class AvataxOrderConfirmedPayloadTransformer {
       orderId: order.id,
     });
     const customerCode = customerCodeResolver.resolveOrderCustomerCode(order);
+    const billingAddress = order.billingAddress;
+
+    if (!billingAddress) {
+      throw new Error("Billing address not found in OrderConfirmed subscription.");
```

Comment:
suggestion: I think we should use a ModernError subclass here instead of generic `Error`

Reply:
I agree, but it hasn't been installed in the app, and I would rather not spend time on it. Feel free to pop it in